variables.tf

```
@@ -1,22 +1,22 @@
 variable "environment" {
-  description = "An environment qualifier for the resources this module creates, to support a Terraform SDLC. The environment value is also the tag value that secrets must be tagged with to be accessible by this addon."
-  type        = string
-}
-
-variable "tag_name" {
-  description = "Name of the tag that resources must be tagged with to be accessible by this addon."
-  default     = "SymEnv"
+  description = "The unique name of the environment in which you are deploying this AWS IAM Policy (e.g. staging, or prod)."
   type        = string
 }
 
 variable "tags" {
-  description = "Additional tags to apply to resources"
+  description = "Additional tags to apply to resources."
   type        = map(string)
   default     = {}
 }
 
 variable "stream_arns" {
-  description = "List of Data Stream ARNs allowed to be accessed by the Sym Runtime."
+  description = "List of Data Stream ARNs allowed to be published to by the Sym Runtime."
```

Reads a little weird, maybe slight rewording?
```suggestion
  description = "List of Data Stream ARNs the Sym Runtime is allowed to publish to."
```